src/Search/Api/SearchResultDiscriminator.php

```diff
@@ -12,6 +12,7 @@
 final class SearchResultDiscriminator implements EventSubscriberInterface
 {
     public static $articleTypes = [
+        'article',
```

Comment:
Okay so just so I'm clear:

SQS sends `{type: article}` and the real data is `{type: research-article}` or any of the others in the list.